src/trans.cpp

```diff
@@ -33,6 +33,9 @@
 #include "proj_internal.h"
 #include <math.h>
 
+#include <cmath>
```

Comment:
[minor] There is an include <math.h> a few lines above. Is there any difference?

Reply:
yes, the scripts check that if std::isnan / isinf / isfinie is used, cmath is included.  Including math.h itself will not bring those symbols. I suspect cmath is already included indirectly here.